src/TRoomDB.cpp

```diff
@@ -715,15 +715,15 @@ void TRoomDB::auditRooms(QHash<int, int>& roomRemapping, QHash<int, int>& areaRe
     if (!missingAreasNeeded.isEmpty()) {
         if (mudlet::self()->showMapAuditErrors()) {
             QString alertMsg = tr("[ ALERT ] - %n area(s) detected as missing in map: adding it/them in.\n"
-                                  "Look for further messages related to the rooms that are supposed\n"
-                                  "to be in this/these area(s)...",
+                                  " Look for further messages related to the rooms that are supposed\n"
+                                  " to be in this/these area(s)...",
```

Comment:
Those leading spaces are not helpful. They put off the hanging paragraph formatting for the `[ XXX ] -` messages (I can't recall but I think the `cTelnet::postMessage(...)` handler may strip off leading spaces of continuation lines, but it certainly adds 8 of them itself at the beginning of all but the first line to try and force that style of paragraph formatting
```suggestion
                                  "Look for further messages related to the rooms that are supposed\n"
                                  "to be in this/these area(s)...",
```

You may wish to action this.

Reply:
Removed.